### PR TITLE
CC-1698 - Shutdown takes too long

### PR DESCRIPTION
**Pull request by morr-zenoss**
Address CC-1698 - Shutdown takes too long

This PR includes a couple of changes JP suggested to help with shutdown. 
The primary fix is non-code: make sure you're using lvm thin pool storage, and not loopback.

### CODE REVIEW

File: cli/api/daemon.go [hunk 1]
```diff
@@ -361,7 +361,6 @@ func (d *daemon) run() (err error) {
 	}
 
 	zzk.ShutdownConnections()
-	volume.ShutdownAll()
```
Comment (smousa) [inline cli/api/daemon.go]: Why did you remove this line?

Comment (morr-zenoss) [inline cli/api/daemon.go]: Because JP asked me to - it was causing problems with NFS mounts.
